sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/coding/MaxLineLengthCheck.java

```diff
@@ -39,20 +42,38 @@
     defaultValue = "" + DEFAULT_MAX_LINE_LENGTH)
   public int maxLength = DEFAULT_MAX_LINE_LENGTH;
 
+  private final Set<Integer> ignoredLines = new HashSet<>();
+
   @Override
   public void startDocument(List<Node> nodes) {
+    ignoredLines.clear();
```

Comment:
would it work as well if we get rid of overriding `startDocument `and we add this instruction at the end of `endDocument`?